rdt-open-reader/src/test/java/io/ona/rdt/robolectric/fragment/RDTJsonFormFragmentTest.java

```diff
@@ -148,10 +147,8 @@ public void testBackClickShouldShowConfirmationDialog() {
     }
 
     @Test
-    public void testNavigationShouldNavigateToCorrectStep() {
-        ReflectionHelpers.callInstanceMethod(jsonFormFragment, "initializeBottomNavigation",
-                ReflectionHelpers.ClassParameter.from(JSONObject.class, new JSONObject()),
-                ReflectionHelpers.ClassParameter.from(View.class, jsonFormFragment.getRootLayout()));
+    public void testNavigationShouldNavigateToCorrectStep() throws Exception {
```

Comment:
Why is this change needed?

Reply:
https://github.com/onaio/rdt-standard/pull/656#discussion_r554880097

Reply:
Could you revert this change?

Reply:
reverted